components/RuiAccordion/src/RuiAccordion.ts

```diff
@@ -58,21 +162,20 @@ export class RuiAccordion extends LitElement {
 		this._items.forEach(element => {
 			element.open = false;
 		});
-
 		event.target.open = true;
 	};
 
 	/**
 	 * Render method
+	 * @returns {TemplateResult}
 	 */
 	public render(): TemplateResult {
-		const buttonCollapse = this.querySelector('[slot=button-collapse]');
-		const buttonExpand = this.querySelector('[slot=button-expand]');
-
 		return html`
 			<div class="outer-container">
-				<slot name="heading"></slot>
-				${buttonCollapse || buttonExpand
+				<div id="accordion-heading">
```

Comment:
Lets put this into a `renderHeading` method. I've started doing this so if consumers want to override just one section they can without having to override the entire `render` method